ui/pages/confirmations/components/gas-details-item/gas-details-item.test.js

```diff
@@ -203,4 +203,51 @@ describe('GasDetailsItem', () => {
       expect(screen.queryAllByText('ETH').length).toBeGreaterThan(0);
     });
   });
+
+  it('does not render icon if network is not busy', async () => {
+    await render();
+    await waitFor(() => {
+      expect(
+        screen.queryByTestId('network-busy-tooltip'),
+      ).not.toBeInTheDocument();
+    });
+  });
+
+  it('renders icon if network is busy', async () => {
+    await render({
+      contextProps: {
+        gasFeeEstimates: {
+          high: {
+            suggestedMaxPriorityFeePerGas: '1',
+          },
+        },
+        gasFeeEstimatesByChainId: {
+          ...mockState.metamask.gasFeeEstimatesByChainId,
+          '0x5': {
+            gasFeeEstimates: {
+              high: {
+                suggestedMaxPriorityFeePerGas: '1',
+              },
+              networkCongestion: 0.7,
+            },
+          },
+        },
+        transaction: {
+          txParams: {
+            gas: '0x52081',
+            maxFeePerGas: '0x38D7EA4C68000',
+          },
+          userFeeLevel: 'medium',
+          dappSuggestedGasFees: {
+            maxPriorityFeePerGas: '0x38D7EA4C68000',
+            maxFeePerGas: '0x38D7EA4C68000',
+          },
+        },
+      },
+    });
+
+    await waitFor(() => {
+      expect(screen.queryByTestId('network-busy-tooltip')).toBeInTheDocument();
+    });
+  });
```

Comment:
Can you also add a unit test for a legacy transaction?

Reply:
oh yeah yeah.. but I meant add a test where `transaction.txParams.type` is set to `0x0` and ensure that the `network-busy-tooltip` does not appear.

Reply:
ต้องการทดสอบ###